pincer/core/ratelimiter.py

```diff
@@ -0,0 +1,132 @@
+# Copyright Pincer 2021-Present
```

Comment:
Imo it would be better in like http it doesn’t really need its own file but idk

Reply:
No way. Http is already 500 lines which is close to really hard to work with imo.